### PR TITLE
Support configure option '--with-openssl=fetched' on Windows

### PR DESCRIPTION
**Pull request by keithc-ca**
Issue: eclipse-openj9/openj9#14302.

### CODE REVIEW

File: closed/openssl.gmk [hunk 1]
```diff
@@ -18,50 +18,65 @@
...
+  ifeq ($(OPENJDK_TARGET_CPU), ppc64le)
+    OPENSSL_TARGET := linux-ppc64le
+  else ifeq ($(OPENJDK_TARGET_CPU), s390x)
+    OPENSSL_TARGET := linux64-s390x
+  else ifeq ($(OPENJDK_TARGET_CPU), x86_64)
+    OPENSSL_TARGET := linux-x86_64
+  endif
+else ifeq ($(OPENJDK_TARGET_OS), macosx)
+  ifeq ($(OPENJDK_TARGET_CPU), arm64)
+    OPENSSL_TARGET := darwin64-x86_64-cc
```
Comment (pshipton) [inline closed/openssl.gmk:64]: This specifies x86 instead of arm.

Comment (keithc-ca) [reply closed/openssl.gmk:64]: Good catch. I inverted that 'ifeq' element badly.

Comment (keithc-ca) [reply closed/openssl.gmk:64]: I plan to replace the series of CPU cases to:
```
  ifneq (,$(filter arm64 x86_64, $(OPENJDK_TARGET_CPU)))
    OPENSSL_TARGET := darwin64-$(OPENJDK_TARGET_CPU)-cc
  endif
```
and apply a similar change for Linux.

**Comment by keithc-ca** [comment]
Fixed calculation of `OPENSSL_TARGET`.

**Comment by keithc-ca** [comment]
Updated calculation of `OPENSSL_TARGET`; removed unnecessary include of `MakeBase.gmk`.

**Review by pshipton — APPROVED** [review]

**Pull request merged**